training/src/main/scala/com/airbnb/aerosolve/training/LowRankLinearTrainer.scala

```diff
@@ -217,6 +230,30 @@ object LowRankLinearTrainer {
     TrainingUtils.saveModel(model, config, key + ".model_output")
   }
 
+  private def normalizeWeightVectors(model: LowRankLinearModel, maxNorm: Double) = {
```

Comment:
capWeightVector? but this should ideally call a capNorm function in FloatVector so you don't duplicate code twice